schematools/importer/__init__.py

```diff
@@ -15,14 +15,16 @@
     "date": Date,
     "time": Time,
     "date-time": DateTime,
+    "uri": String, # aangepast,
+    "email": String, # aangepast,
 }
 
 JSON_TYPE_TO_PG = {
     "string": String,
     "object": String,
     "boolean": Boolean,
     "integer": Integer,
-    "number": Float,
+    "number": Float,    
```

Comment:
Klein dingetje. Ik zie vaak in de code trailing spaces geintroduceerd worden. Dat leidt tot een soort van vals-positieve diffs. Goede gewoonte is om deze trailing spaces on-save te verwijderen. Sommige editors kunnen dat, vscode ook zo te zien: https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut

Reply:
Thnx, werkt goed, alle trailing white spaces worden bij file save verwijderd :-)